extensions/samples/electron-webpack/index.html

```diff
@@ -1,12 +1,38 @@
 <!DOCTYPE html>
-<html>
- <head>
-   <meta charset="UTF-8" />
-   <title>Sample Electron Msal Node Etensions</title>
- </head>
- <body>
-   <mgt-login></mgt-login>
-   <mgt-agenda group-by-day></mgt-agenda>
-   <script type="module" src="./dist/renderer.js"></script>
- </body>
-</html>
\ No newline at end of file
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no"> 
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self'" />
+    <title>MSAL Node Electron Sample App</title>    
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+      <a class="navbar-brand">MS Identity Platform</a>
+      <div class="btn-group ml-auto dropleft">
+          <button type="button" id="SignIn" class="btn btn-secondary" aria-expanded="false">
+            Sign In
+          </button>
+          <button type="button" id="SignOut" class="btn btn-success" hidden aria-expanded="false">
+            Sign Out
+          </button>
+      </div>
+    </nav>
+    <br>
+    <h5 class="card-header text-center">Electron App Calling Microsoft Graph using MSAL Node</h5>
+    <br>
+    <div class="row" style="margin:auto" >
```

Comment:
Might want to indent some of these for readability.

Reply:
Done